src/rules/selector-combinator-space-after/index.js

```diff
@@ -44,7 +44,7 @@ export function selectorCombinatorSpaceChecker({ locationChecker, root, result,
       styleSearch({
         source: selector,
         target: _.toArray(nonSpaceCombinators),
-        functionArguments: "skip",
+        parentheticals: "skip",
```

Comment:
> What about `a:not(li + a)`, in which a real combinator is inside parentheses?

I hadn't realised, up until now, that `:not` can take more than just a single *simple selector* in [Selectors Level 4](https://drafts.csswg.org/selectors-4/#negation). Good catch.

> Any way we can check that?

Even before this PR any combinators within pseudo classes were skipped. So, I think checking them is a breaking change for `future-major`. Shall I can create a new issue for that?